specification/devcenter/DevCenter/service.tsp

```diff
@@ -1,9 +1,6 @@
 import "@typespec/rest";
 import "@typespec/http";
 import "@azure-tools/typespec-azure-core";
-import "./DevBox/routes.tsp";
-import "./DevCenter/routes.tsp";
-import "./Environments/routes.tsp";
```

Comment:
Let's not revert the improvements that were made in the last typespec migration in this PR where we imported every route into the central service definition.

Reply:
Undone. There were temp changes to make reviewing the breaking changes a bit easier, it's back to normal now